fem/gridfunc.hpp

```diff
@@ -141,24 +141,70 @@ class GridFunction : public Vector
 
    void GetVectorValue(int i, const IntegrationPoint &ip, Vector &val) const;
 
+   double GetBdrValue(int i, const IntegrationPoint &ir, int vdim = 1) const;
```

Comment:
Can you add short comments to help users understand the difference between e.g. `GetBdrValue` and `GetBdrFaceValue`?

Along these lines, is there a way to consolidate some of these methods?

Reply:
Sure, I added comments in 4d7147b and added a note to `CHANGELOG`. Some consolidation seems to be also possible to smaller extend, I will look at it.

Reply:
I consolidated Get(Bdr)Value(s) methods in 5b5821d. It saves just a few lines, but it is cleaner this way.